PhysicsTools/PatAlgos/python/slimming/slimmedMuons_cfi.py

```diff
@@ -15,6 +15,19 @@
     segmentsMuonSelection = cms.string("pt > 50"), #segments are needed for EXO analysis looking at TOF and for very high pt from e.g. Z' 
     saveSegments = cms.bool(True),
     modifyMuons = cms.bool(True),
-    modifierConfig = cms.PSet( modifications = cms.VPSet() )
+    modifierConfig = cms.PSet( modifications = cms.VPSet() ),
+    trackExtraAssocs = cms.VInputTag(["muonReducedTrackExtras", "slimmedMuonTrackExtras"]),
 )
 
+# full set of track extras not available in existing AOD
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+run2_miniAOD_80XLegacy.toModify(slimmedMuons, trackExtraAssocs = ["slimmedMuonTrackExtras"])
+
+from Configuration.Eras.Modifier_run2_miniAOD_94XFall17_cff import run2_miniAOD_94XFall17
+run2_miniAOD_94XFall17.toModify(slimmedMuons, trackExtraAssocs = ["slimmedMuonTrackExtras"])
+
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+pp_on_AA_2018.toModify(slimmedMuons, trackExtraAssocs = ["slimmedMuonTrackExtras"])
+
+from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
+run2_miniAOD_UL.toModify(slimmedMuons, trackExtraAssocs = ["slimmedMuonTrackExtras"])
```

Comment:
`(run2_miniAOD_80XLegacy | run2_miniAOD_94XFall17 | pp_on_AA_2018 | run2_miniAOD_UL).toModify(...`
will be shorter and more maintainable

it seems like a repeated pattern

Reply:
This comment wasn't taken into account yet.
It is just a simplification, but I'd suggest to consider it already in this PR, for an easier maintenance

Reply:
This was addressed by https://github.com/cms-sw/cmssw/pull/31217/commits/17ae4b93297f1340df88ebf138ad7ca877541fb1 no?

Reply:
> This was addressed by [17ae4b9](https://github.com/cms-sw/cmssw/commit/17ae4b93297f1340df88ebf138ad7ca877541fb1) no?

Ah, ok! Why I still did not see it? Maybe I had open an outdated version on my github...
Now I can see the modification applied, sorry for the noise